### DISCUSSION TITLE
How can i attach SqlLocalDB with squirrel installer

### DISCUSSION THREAD

**Discussion by msham2022** [Squirrel/Squirrel.Windows#1812] [Q&A] [answered]
How can i attach SqlLocalDB with squirrel installer as i wanted to install SqlLocalDB prerequisites

**Comment by owns** [comment]
I'd recommend making your app squirrel aware and do check/install then. First draft could be has simple have a prompt to install the dependency and close.

**Comment by anaisbetts** [comment, marked as answer]
This isn't supported by Squirrel. Any installation that requires user interaction or UAC prompts will not work.
